test/test_mps.py

```diff
@@ -1458,6 +1458,58 @@ def helper(shape, ks, padding=0, dilation=1, ceil_mode=False, return_indices=Fal
             helper((1, 1000, 1, 4), ks=(1, 4), ceil_mode=True,
                    return_indices=True, test_ties=test_ties)  # test for max_pool1d
 
+    # Test forward maxpool2d
+    def test_max_pool3d(self):
+        def helper(shape, ks, padding=0, dilation=1, ceil_mode=False, return_indices=False, test_ties=False):
+
+            cpu_x = None
+            if (test_ties):
```

Comment:
This is Python, not C++
```suggestion
            if test_ties:
```